modules/logging-release-notes-5-8-15.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-15_{context}"]
+= Logging 5.8.15
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10052[RHBA-2024:10052] and link:https://access.redhat.com/errata/RHBA-2024:140310[RHBA-2024:140310].
+[id="logging-release-notes-5-8-15-bug-fixes_{context}"]
+== Bug fixes
+*  Before this update, Loki did not correctly load some configurations, which caused issues when using Alibaba Cloud or IBM Cloud object storage. This update fixes the configuration-loading code in Loki, resolving the issue. (link:https://issues.redhat.com/browse/LOG-6294[LOG-6294])
+* Before this update, upgrades to version 6.0 failed with errors if a Log File Metric Exporter instance was present. This update fixes the issue, enabling upgrades to proceed smoothly without errors. (link:https://issues.redhat.com/browse/LOG-6328[LOG-6328])
+[id="logging-release-notes-5-8-15-CVEs_{context}"]
+== CVEs
```

Comment:
In the preview, **== CVEs** appears to be on the wrong line. It's tucked in after the bug link and should likely be on its own line following the bug link.

Reply:
Good eyes, thanks!